src/index.js

```diff
@@ -332,6 +333,8 @@ function addSlotToRoot (root, slot) {
   }
 
   rootToSlotMap.get(root)[slotName] = slot;
+  !slotToRootMap.has(slot) && slotToRootMap.set(slot, root);
```

Comment:
I think it might not be migrated and still in JSCS - don't worry about it, it is not a blocker for this PR I think, I was just surprised to see it.